sonar-plugin-api/src/main/java/org/sonar/api/batch/rule/Rule.java

```diff
@@ -51,19 +50,12 @@
   RuleStatus status();
 
   /**
-   * Sub characteristic key.
-   *
-   * @since 4.3
-   */
-  @CheckForNull
-  String debtSubCharacteristic();
```

Comment:
you delete a method from a class in package api without deprecating it first? is that really ok?